contracts/src/Teleporter/upgrades/README.md

```diff
@@ -79,18 +79,34 @@ To prevent anyone from calling the dapp's `updateMinTeleporterVersion`, which wo
     }
 ```
 
-For sending messages with the Teleporter registry, dapps should generally use `TeleporterRegistry.getLatestTeleporter` for the latest version, but if the dapp wants to send a message to a specific version, it can use `TeleporterRegistry.getTeleporterFromVersion` to get the specific Teleporter version.
+For sending messages with the Teleporter registry, dapps should generally use `TeleporterUpgradeable._getTeleporterMessenger`, which by default will use the latest version. If the dapp wants to send a message through a specific Teleporter version, it can override `_getTeleporterMessenger()` to use the specific Teleporter version with  `TeleporterRegistry.getTeleporterFromVersion`.
```

Comment:
Can you add here how TeleporterUpgradeable._getTeleporterMessenger extends TeleporterRegistry.getLatestTeleporter?